controllers/todo.js

```diff
@@ -1,33 +1,247 @@
 const { ToDo, Token } = require("../models");
+const todo = require("../models/todo");
 
 // All the given method require token.
 // So be sure to check for it before doing any stuff
 // HINT: Create a middleware for above :)
 
+//ok lets create a middleware XD
+const verification = (req, res) => {
+  try{
+    const token = req.body.token || req.query.token ||
+                  req.headers["x-access-token"] || req.headers.token ||
+                  req.headers.authorization.split(' ')[1];
+    if (!token)
+      return false;
+    else
+      return token;
+  }
+  catch(err){
+    return false;
```

Comment:
be specific with the message. If i dont give any token, I should get `no token provided`. If token is given, but invalid then message should be `invalid token given`.